src/postgresql/pgolap.tcl

```diff
@@ -50,16 +50,31 @@ pg_result $result -clear
 return
 }
 
-proc ConnectToPostgres { host port user password dbname } {
+proc ConnectToPostgres { host port azure user password dbname } {
+    #Azure requres machine name suffix when login
+    if { $azure eq "true" } {
+        set contains_machine_name [string match "*@*" $user]
+        if {$contains_machine_name == 0} {
+            set machine [lindex [split  "$host"  .] 0]
+            set user "$user@$machine"
+        }
+    }
+
+    #Azure requires ssl connection
+    set sslConnectionEnabled 0
```

Comment:
Do we want to always disable SSL?

Reply:
requiressl is a deprecated option in favour of sslmode along with an number of additional SSL options. 
All options can be seen with:
`foreach prop [ pg_conndefaults ] { puts $prop } `
If SSL support is required for PostgreSQL then I would suggest that this be a separate task and not tied to Azure in particular. If this is needed I'd suggest adding an Issue for adding SSL Support to PostgreSQL as a starting point. There are connections in multipl places include the standard build and driver script but also the asynchronous connections and connect pooling and any ssl option would be to be selectable in both CLI and GUI.